web/src/app/components/app-board-form/app-board-form.component.html

```diff
@@ -0,0 +1,17 @@
+<ng-container *ngIf="showForm; else addCta;">
```

Comment:
we should reduce as much `ngIf`

Reply:
the component containing it removed from this pr